### DISCUSSION TITLE
Tarteaucitron et matomo

### DISCUSSION THREAD

**Discussion by DouzDevp** [AmauriC/tarteaucitron.js#714] [Q&A]
Bonjour, 

Dans le cadre de nos projets de fin d'étude, nous avons développé 2 sites. Sur chacun nous avons installé tarteaucitron (au top!) et nous avons installé matomo comme système de tracking.
Nous avons suivi le guide d'installation sur tarteaucitron.io mais nous avons deux pépins :
- La popup pour recueillir le consentement ne s'affichent pas à la première visite sur les sites
- Les cookies matomo sont automatiquement enregistrés, tarteaucitron indique true automatiquement au navigateur)  

Nous avons effectué d'autres testes avec d'autres services et tout fonctionne alors correctement. Il y a t il un problème avec la gestion de matomo par tarteaucitron ? Avez-vous une idée ?

Note : Le premier projet est développé sur symfony, et le deuxième est développé from scratch. Et nous avons bien reglé le hightprivacy sur true !

**Comment by nicozerr** [comment]
Bonjour,

Je vous invite à lire #694 et les liens associés dans les commentaires. Nous ne sommes pas tous tout à fait d'accord et la CNIL n'aide pas beaucoup.

Personnellement, avec Matomo hébergé sur un serveur personnel et non pas sur le Cloud de Matomo, avec l'IP anonymisé (configuré ainsi par défaut), et avec la non-prorogation des cookies de tracking, mon DPD considère que Matomo peut être déployé sans consentement.

Il y a encore peu de temps, la CNIL considérait Matomo comme exempté de consentement. Tarteaucitron a donc été configuré pour activer Matomo par défaut.
Aujourd'hui, au regard des règles à suivre pour en être exempté, et même après la récente mise à jour du RGPD, Matomo semble toujours exempté.

**Reply by DouzDevp** [reply]
Bonjour,
Merci pour votre réponse, je n'avais pas vu cette discussion. Effectivement le débat est intéressant ! :)
